src/cases/BaseTestSuite.php

```diff
@@ -5,6 +5,8 @@
 use luya\Boot;
 use luya\testsuite\fixtures\ActiveRecordFixture;
 use Yii;
+use Yoast\PHPUnitPolyfills\Polyfills\AssertStringContains;
```

Comment:
No need for this `use` statement, nor for the `use AssertStringContains;` trait import below.

The `Yoast\PHPUnitPolyfills\TestCases\TestCase` which you are using already includes all relevant polyfill traits.